pkg/plugin/v3/scaffolds/internal/templates/config/api/spoke_conversion.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &SpokeConversion{}
+
+type SpokeConversion struct { // nolint:maligned
+	file.TemplateMixin
+	file.MultiGroupMixin
+	file.BoilerplateMixin
+	file.ResourceMixin
+
+	SpokeVersion string
+}
+
+func (c *SpokeConversion) SetTemplateDefaults() error {
+	c.Resource.Version = c.SpokeVersion
+	if c.Path == "" {
+		if c.MultiGroup {
+			c.Path = filepath.Join("apis", "%[group]", "%[version]", "%[kind]_conversion.go")
+		} else {
+			c.Path = filepath.Join("api", "%[version]", "%[kind]_conversion.go")
+		}
+	}
+	c.Path = c.Resource.Replacer().Replace(c.Path)
+	c.TemplateBody = spokeConversionTemplate
+	c.IfExistsAction = file.Error
+
+	return nil
+}
+
+const spokeConversionTemplate = `{{ .Boilerplate }}
+
+package {{ .Resource.Version }}
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+
+func (src  *{{ .Resource.Kind }}) ConvertTo(dstRaw conversion.Hub) error {
+	// Implement logic to convert from hub to spoke version.
```

Comment:
```suggestion
	// Implement your logic here to convert from hub to spoke version.
```

Reply:
To be similar to: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/controllers/admiral_controller.go#L44